models/echo/README.md

```diff
@@ -0,0 +1,28 @@
+# Echo with Feedback
+This design showcases a simple echo audio effect with a feedback loop based off the Mathworks example [Delay-Based Audio Effects](https://www.mathworks.com/help/audio/ug/delay-based-audio-effects.html).
+
+## Requirements
```

Comment:
I agree. Rather than rehashing the steps over and over, we should get a standard short boilerplate that links them to the getting started guides in the docs repo

Reply:
Resolved with 5b55ca6